sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerClientConfiguration.java

```diff
@@ -37,58 +40,87 @@
 @EnableConfigurationProperties({ AADAuthenticationProperties.class })
 @ConditionalOnClass({ BearerTokenAuthenticationToken.class, OAuth2LoginAuthenticationFilter.class })
 @ConditionalOnProperty(prefix = "azure.activedirectory", value = "client-id")
-public class AADResourceServerOboConfiguration {
+public class AADResourceServerClientConfiguration {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(AADResourceServerOboConfiguration.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(AADResourceServerClientConfiguration.class);
 
     @Autowired
     private AADAuthenticationProperties properties;
 
     @Bean
     @ConditionalOnMissingBean({ ClientRegistrationRepository.class })
     public ClientRegistrationRepository clientRegistrationRepository() {
-        final List<ClientRegistration> oboClients = createOboClients();
+        final List<ClientRegistration> oboClients = createClients();
```

Comment:
the local variable is still named `oboClients`, I suppose you also want to change this.